gradle/errorprone.gradle

```diff
@@ -17,6 +17,8 @@ dependencies {
 
     //taken from https://github.com/google/error-prone/releases
     errorprone 'com.google.errorprone:error_prone_core:2.5.1'
+    //needed to compile on the new F-Droid virtual machines
+    errorproneJavac 'com.google.errorprone:javac:9+181-r4173-1'
```

Comment:
This is only needed if we are using JDK8 as the compiler. Is there a way to tell fdroid to use the installed OpenJDK11?

Reply:
It's very possible I accidentally used JDK8. I'll do more tests and change this commit when I'm done.

As far as I can see, [there's no option to select JDK](https://f-droid.org/en/docs/Build_Metadata_Reference/), but you can always install it manually using the `sudo` parameter. That's what my metadata file above is supposed to do:

`sudo: apt-get install -y openjdk-11-jdk-headless openjdk-11-jre-headless`

If that is succesfully installed, it's supposed to run `update-alternatives` so that `java` and `javac` point to the new version. But that command can fail in several different ways, so I'll try uninstalling the old JDK first.

Reply:
Confirmed, my PR wasn't needed after all. I was compiling with JDK8.

I'll post the updated F-Droid metadata below.